erpnext/manufacturing/doctype/work_order/work_order.json

```diff
@@ -182,7 +182,6 @@
    "reqd": 1
   },
   {
-   "default": "1.0",
```

Comment:
This was intentionally added a while back.

Reply:
I didn't remove it I just disabled the set only once property and fixed sider issues that's it

Reply:
Maybe you had an older local version of the file locally hence it appears to be removed. 

The change still needs to be undone though.

Reply:
ok